code/PostgreSQLSchemaManager.php

```diff
@@ -1471,4 +1472,25 @@ public function set($values)
         user_error("PostGreSQL does not support multi-enum", E_USER_ERROR);
         return "int";
     }
+
+    /**
+     * Update existing enum value that have been invalidated
+     * @param string $specValue Updated specification for the column.
+     * @param string $spec_orig Original specification for the column.
+     * @param string $fieldValue Current specification for the column.
+     * @param string $table Name of the table.
+     * @param string $field Name of the field.
+     * @return Query|null
+     */
+    protected function resetInvalidatedEnumValue($specValue, $spec_orig, $fieldValue, $table, $field)
```

Comment:
We should not rely on the incoming arguments not to be invalid or even malicious.
Currently `$field`, `$table`, `$inValues` or `$default` may be crafted to do some bad stuff.
We must use `preg_quote` for the regex, `pg_escape_literal` for values and `pg_escape_identifier` for table and field names.
Ideally `$inValues` has to be split up and its every single value should be escaped as well.

Reply:
I'll have a look, but in practice all the values passed to `resetInvalidatedEnumValue` will have been generated by DBSchemaManager itself.

But I agree it's good practice to minimise those assumptions if we can.